playbooks/openstack/sample-post-install-playbooks/add-rhn-pools.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: cluster_hosts
+  vars:
+    rhn_pools: []
+  tasks:
+  - name: Attach additional RHN pools
```

Comment:
this prolly makes more sense to be configured as pre-install (post-provision) than post install?